src/components/pages/berthPage/ReleaseNote.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { FormattedHTMLMessage, FormattedMessage } from 'react-intl';
+import { Col, Container, Row } from 'reactstrap';
+
+import './releaseNote.scss';
+
+const ReleaseNote = () => (
```

Comment:
I think something like `WinterStorage2019Warning` would be a better name for this component since it looks like a one-timer without any re-usability.

Reply:
Yes, you're right, that makes sense.